images/build/cli.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM oryxdevmcr.azurecr.io/public/oryx/build-slim:latest AS build
```

Comment:
do we intend to build from the oryxdevmcr build-slim image? or we should also consider to build it from production

Reply:
The intention of the CLI image is to only have the artifacts needed to run the `oryx` CLI -- since we're copying the build artifacts from the Oryx project to the new image, I don't think it matters if we're using `slim` or `latest`. Since this image contains only the bare minimum needed to run the CLI, specifically the `oryx dockerfile` command (which doesn't need any dev tools), we want to keep the image as small as possible and copy over just the artifacts, another reason I don't believe it matters which tag we're using. Let me know what you think.